src/content/quickstart/vsphere_aws/_index.md

```diff
@@ -0,0 +1,176 @@
+---
+date: 2020-02-21T13:36:18+01:00
+title: "OpenShift with Service Discovery (vSphere/AWS)"
+weight: 15
+---
+
+In this quickstart guide, we shall cover the necessary steps to deploy OpenShift Container Platform (OCP) on vSphere and AWS. Once the OCP clusters are deployed, we shall cover how to deploy Submariner and connect the two clusters.
+
+### OpenShift Prerequisites
```

Comment:
Can we use includes from pre-existing content like here? 

https://github.com/submariner-io/submariner-website/blame/master/src/content/quickstart/openshiftgn/_index.md#L7

Reply:
Yes we need to consolidate all we can :) I agree the "aws" part does not belong there, but the other parts.

I agree on consolidating this later.